core/src/main/scala/caliban/schema/Schema.scala

```diff
@@ -81,13 +81,7 @@ trait Schema[-R, T] { self =>
    */
   def arguments: List[__InputValue] = Nil
 
-  /**
-   * Boolean flag indicating whether the [[resolve]] method should be wrapped in a [[FunctionStep]] when resolving
-   * object fields.
-   *
-   * Should be `false` except for objects or wrappers of objects. This prevents fields of case classes to be resolved when they're not
-   * requested in the query, which significantly improves performance in larger schemas
-   */
+  @deprecated("No longer needed")
```

Comment:
Shall I just delete this method since it's package private? Next release will likely be 1.5.0 given the play and zio-http changes

Reply:
ok with me